src/commands/general/birthday/register.ts

```diff
@@ -107,7 +62,7 @@ export class ListCommand extends Command {
 		const { guildId, memberPermissions } = interaction;
 		const authorIsTarget = interaction.user.id === targetUser.id;
 
-		if (!authorIsTarget && !memberPermissions.has('ManageRoles')) {
+		if (!authorIsTarget && !memberPermissions.has('ManageGuild')) {

```

Comment:
Revert to `ManageRoles`. This is intended behavior.
```suggestion
		if (!authorIsTarget && !memberPermissions.has('ManageRoles')) {
```